### DISCUSSION TITLE
Omada controller not updating to latest version

### DISCUSSION THREAD

**Discussion by istvanSA** [tteck/Proxmox#3314] [Help Needed]
The omada update says it completed succesfully but its installing the same version and nog v5.14.20.9 which is the latest version

Updating Omada Controller
(Reading database ... 21180 files and directories currently installed.)
Preparing to unpack Omada_SDN_Controller_v5.13.30.8_linux_x64.deb ...
Omada Controller is running, going to stop it.
Stopping Omada Controller 
Stop successfully.
A previous controller version has been detected.
Unpacking omadac (5.13.30.8) over (5.13.30.8) ...
Setting up omadac (5.13.30.8) ...
Install Omada Controller succeeded!

**Comment by istvanSA** [comment]
hmz, seems its a nvm and the windows version is newer then the latest linux version. Thats strange

**Comment by bigeuro** [comment]
I ran script to update Omada, and it didn't update to latest. Omada 5.14.26.1 is available for Linux, but looks like script doesn't bring it up to that version.

**Reply by tteck** [reply]
According to https://www.tp-link.com/us/support/download/omada-software-controller/ the Linux version is v5.13.30.8

![Screenshot 2024-07-09 11 20 18 AM](https://github.com/tteck/Proxmox/assets/21342670/9006b2db-9a7c-403c-8b52-fb03bc07618f)

**Reply by bigeuro** [reply]
Yeah, I see that. I guess it's an issue on TPlink's end moving the 5.14.x build into official/stable release for linux.

**Reply by bigeuro** [reply]
After make a post asking why the Linux 5.14.x version wasn't official, their devs made it official. I ran your script and I'm updated. Thanks again for your amazing work.

**Comment by tismofied** [comment]
So the wise thing to do here is wait to update until it's sorted out right?

**Reply by bigeuro** [reply]
Well, I made a post in Omada Community forum, and they made the new 5.14.x Linux official. I ran the proxmox helper script upgrade script within the LXC console and I'm now on the 5.14 version!
